shared/profile/add-to-team/container.tsx

```diff
@@ -41,7 +40,7 @@ type State = {
   selectedTeams: SelectedTeamState
 }
 
-class AddToTeamStateWrapper extends React.Component<ExtraProps & AddToTeamProps, State> {
+export class AddToTeamStateWrapper extends React.Component<ExtraProps & AddToTeamProps, State> {
```

Comment:
why export this?

Reply:
Ts wants it due to how the external modules stuff works. Not expecting it will give you an error